cmake/FindSampleRate.cmake

```diff
@@ -1,4 +1,4 @@
-# Findsamplerate.cmake
+# FindSampleRate.cmake
```

Comment:
This is a VERY BAD IDEA! Windows git gets mightily confused when a file is renamed so that only its case changes. Unless it has been fixed recently anyone pulling this commit on Windows will end up with a failed pull and a broken tree and fixing it is not easy. When we hit this we just told everyone to pull again and not do this again.

Reply:
Okay, I restored the old name, it seems to work. It's weird that CMake is sometimes case sensitive and sometimes not.

Reply:
This needs to be squashed out of the history so no-one hits it while bisecting. Otherwise looks reasonable, has it been build-tested on Windows?

Reply:
> has it been build-tested on Windows?

Yes, I successfully built it with MSVC + vcpkg and MSYS2.